SU-SWS/stanford_profile
# D8CORE-633: add files from stanford_module_example

# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-633_

# Needed By (Date)
- Sprint 


# Urgency
- Fixes broken stuff


# Steps to Test

- Checkout this branch
- Create a basic page with a WYSIWYG, and cards with images that are with and without text.
- Verify you see the cards and WYSIWYG play nicely at all browser widths.

# Affected Projects or Products
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-633

## Related PRs

## More Information
Here's a [demo of how it behaves](https://drive.google.com/drive/u/1/folders/0BzW3JogKJkUEWTBVeVdIUzNEOU0). Let's remember to delete this!
## Folks to notify
@kerri-augenstein

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
